point-of-sale-app/frontend/checkstyle.xml

```diff
@@ -0,0 +1,178 @@
+<?xml version="1.0"?>
+<!--
+ Copyright 2020 Google LLC
```

Comment:
Do you want to go with 2021?

Reply:
Updated to 2021 in https://github.com/GoogleCloudPlatform/anthos-edge-usecases/pull/1/commits/4db247ea9df66e81c2146b74894ee748e4dc06e6. Thanks for catching that!